src/main/groovy/io/micronaut/build/MicronautDependencyUpdatesPlugin.groovy

```diff
@@ -37,47 +31,14 @@ class MicronautDependencyUpdatesPlugin implements Plugin<Project> {
         }
 
         project.with {
-            tasks.named("dependencyUpdates") {
-                onlyIf {
-                    gradle.taskGraph.hasTask(":useLatestVersions")
-                }
-                checkForGradleUpdate = true
-                gradleReleaseChannel = "current"
-                checkConstraints = true
-                revision = "release"
-                rejectVersionIf { mod ->
-                    mod.candidate.version ==~
-                            micronautBuildExtension.dependencyUpdatesPattern ||
-                            ['alpha', 'beta', 'milestone', 'rc', 'cr', 'm', 'preview', 'b', 'ea'].any { qualifier ->
-                                mod.candidate.version ==~ /(?i).*[.-]$qualifier[.\d-+]*/
-                            } ||
-                            mod.candidate.group == 'io.micronaut' // managed by the micronaut version
-                }
-
-                outputFormatter = { result ->
-                    if (!result.outdated.dependencies.isEmpty()) {
-                        def upgradeVersions = result.outdated.dependencies
-                        if (!upgradeVersions.isEmpty()) {
-                            println "\nThe following dependencies have later ${revision} versions:"
-                            upgradeVersions.each { dep ->
-                                def currentVersion = dep.version
-                                println " - ${dep.group}:${dep.name} [${currentVersion} -> ${dep.available[revision]}]"
-                                if (dep.projectUrl != null) {
-                                    println "     ${dep.projectUrl}"
-                                }
-                            }
-                            throw new GradleException('Abort, there are dependencies to update. Run ./gradlew useLatestVersions to update them in place')
-                        }
-                    }
-                }
-            }
-
-            tasks.named("useLatestVersions") {
-                updateRootProperties = true
+            tasks.register("dependencyUpdates", DeprecatedTask) {
+                it.message.set("The dependencyUpdates task is scheduled for removal")
```

Comment:
Well, the task has actually been removed no?

Reply:
A task is still present, so calling it wouldn't fail, vs completely removing which would cause Gradle to bark.